### ISSUE TITLE
Why the size of validation dataset is 5000 rather than 10,000

### ISSUE THREAD

**Issue by LouiValley** [tjmoon0104/pytorch-tiny-imagenet#5]
"The dataset contains 100,000 images of 200 classes (500 for each class) downsized to 64×64 colored images. Each class has 500 training images, 50 validation images, and 50 test images." _https://towardsdatascience.com/pytorch-ignite-classifying-tiny-imagenet-with-efficientnet-e5b1768e5e8f_

According to this link, I think the dataset for validation is 10,000, but when I print the dataset it is about 5000.
`dataset_sizes = {x: len(image_datasets[x]) for x in ['train', 'val', 'test']}`

**Comment by yangtao2019yt** [comment]
I guess the author remove the original test set (10,000 images). Then cut the original validation set (10,000 images) in half and get two subsets as a new test(5000 images) and validation(5000 images), respectively. The details are in `val_format.py`. 

I also don't understand why only using half of the original validation set as the new test for verifying accuracy in `test_model.py`.

Could the author offer an explanation?

**Comment by tjmoon0104** [comment]
Original dataset test set does not include labels. Which is not usable for experiment purpose.
So I simply divided the validation set (10,000 images) into half 5,000 for validation and 5,000 for test (with labels). 

Validation set was used during training process to calculate validation accuracy and loss. 
At the very end I use the test set to calculate test accuracy.
